frontend/static/js/events.js

```diff
@@ -1,83 +1,126 @@
 function handle__schemespal() {
-    //Google
-    const scriptURLpalquery = 'https://script.google.com/macros/s/AKfycbwgKnPhUBHnlGJ3YudlSeaMfjUe9mPaI-N3Sbz9uar52oDDFf0/exec'
-    const schemespalform = document.getElementById("schemespal")
+  //Google
+  const scriptURLpalquery =
+    "https://script.google.com/macros/s/AKfycbwgKnPhUBHnlGJ3YudlSeaMfjUe9mPaI-N3Sbz9uar52oDDFf0/exec";
+  const schemespalform = document.getElementById("schemespal");
 
-    //filling up the result
-    var segment = $("#filler");
-    segment.html("");
-    $("#filler").hide().append('<div class="text-center"><div class="spinner-border text-primary" role="status"><span class="sr-only">Loading...</span></div></div>').fadeIn(1000);
-    // var url = "https://schemes.sg/schemespredict";
-    var url = '/schemespredict';
+  //filling up the result
+  var segment = $("#filler");
+  segment.html("");
+  $("#filler")
+    .hide()
+    .append(
+      '<div class="text-center"><div class="spinner-border text-primary" role="status"><span class="sr-only">Loading...</span></div></div>'
+    )
+    .fadeIn(1000);
+  // var url = "https://schemes.sg/schemespredict";
+  var url = "http://127.0.0.1:8000/schemespredict";
```

Comment:
call the frontend api
```suggestion
  var url = "/schemespredict";
```